src/GitHub/Data/Definitions.hs

```diff
@@ -226,6 +226,9 @@ data OrgMemberFilter
     deriving (Show, Eq, Ord, Enum, Bounded, Typeable, Data, Generic)
 
 -- | Filter members returned by their role.
+--
+-- This is for querying.
+-- For the response variation, see 'GitHub.Data.Membership.Role'
```

Comment:
Shouldn't that be MembershipRole?

Reply:
yep, renamed that because they're combined into a single namespace.